STM32F405RGTx/Core/Inc/svpwm_thread.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "VHZ.h"
+#include "main.h"
+#include "util.hpp"
+
+#ifndef INC_SVPWM_H_
+#define INC_SVPWM_H_
+
+class SVPWM{
+    public:
+        static float  ModIndex;
+		static float  Freq;
+        static float  SwitchFreq;
+        static float  Alpha;
+        static float  NewEntry;
+        static float  SectorPointer;
+		static float  U;
+        static float  V;
+		static float  W;
+		static float  StepAngle;
+		static float  EntryOld;
+		static float  ta;
+		static float  tb;
+        static float  to;
+

```

Comment:
Is there a reason for this formatting of the fields? If not, can you get them all to be aligned

Reply:
fixed